### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by totoprayogo1916**
**Description**
Update the echo samples

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPdocs
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/general/helpers.rst [hunk 1]
```diff
@@ -92,7 +92,7 @@ use, you'll call it the way you would a standard PHP function.
 For example, to create a link using the ``anchor()`` function in one of
 your view files you would do this::
 
-    <?php echo anchor('blog/comments', 'Click Here');?>
+    <?php echo anchor('blog/comments', 'Click Here'); ?>
```
Comment (kenjis) [inline user_guide_src/source/general/helpers.rst:95]: Remove the last `;`?

File: user_guide_src/source/general/helpers.rst [hunk 2]
```diff
@@ -92,7 +92,7 @@ use, you'll call it the way you would a standard PHP function.
 For example, to create a link using the ``anchor()`` function in one of
 your view files you would do this::
 
-    <?php echo anchor('blog/comments', 'Click Here');?>
+    <?php echo anchor('blog/comments', 'Click Here') ?>
```
Comment (mostafakhudair) [inline user_guide_src/source/general/helpers.rst:95]:
```suggestion
    <?= anchor('blog/comments', 'Click Here') ?>
```

Comment (totoprayogo1916) [reply user_guide_src/source/general/helpers.rst:95]: On it is written, standard coding.

**Comment by MGatner** [comment]
While you're here @totoprayogo1916 would you change the case on booleans? Style Guide specifies they should be "true" and "false".

**Comment by totoprayogo1916** [comment]
> While you're here @totoprayogo1916 would you change the case on booleans? Style Guide specifies they should be "true" and "false".

What does it mean?

I just fixed the example on Userguide, and didn't change anything else.

File: user_guide_src/source/general/helpers.rst [hunk 3]
```diff
@@ -128,7 +128,7 @@ functions::
             {
                 return TRUE;
```
Comment (MGatner) [inline user_guide_src/source/general/helpers.rst:129]: Like this:
```suggestion
                 return true;

```

File: user_guide_src/source/general/helpers.rst (hunk 2 above)
Comment (paulbalandan) [reply user_guide_src/source/general/helpers.rst:95]: I don't get what you mean. You used the short echo tag for `esc` , `set_value`, etc., but not for `anchor`??

File: user_guide_src/source/models/model.rst [hunk 4]
```diff
@@ -100,9 +104,11 @@ The model class has a few configuration options that can be set to allow the cla
 to work seamlessly for you. The first two are used by all of the CRUD methods to determine
 what table to use and how we can find the required records::
 
-    <?php namespace App\Models;
+    <?php
 
-    use CodeIgniter\Model;
+	namespace App\Models;
```
Comment (paulbalandan) [inline user_guide_src/source/models/model.rst:109]: The namespace and use are mis-aligned.

File: user_guide_src/source/general/helpers.rst (hunk 2 above)
Comment (totoprayogo1916) [reply user_guide_src/source/general/helpers.rst]: updated

File: user_guide_src/source/models/model.rst (hunk 4 above)
Comment (totoprayogo1916) [reply user_guide_src/source/models/model.rst]: updated

**Review by paulbalandan — APPROVED** [review]

**Review by MGatner — APPROVED** [review]
Nice work. Thank you @totoprayogo1916

**Pull request merged**